### DISCUSSION TITLE
主机重启，设备配置文件全部丢失，每次都一样

### DISCUSSION THREAD

**Discussion by PWMFI** [AlistGo/alist#4170] [Q&A]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
- [X] I'm sure there are no duplicate issues or discussions.
- [X] I'm sure it's due to `alist` and not something else(such as `Dependencies` or `Operational`).
- [X] I'm sure I'm using the latest version

### Alist Version / Alist 版本

目前最新

### Driver used / 使用的存储驱动

.

### Describe the bug / 问题描述

主机重启，设备配置文件全部丢失，每次都一样

### Reproduction / 复现链接

-

### Logs / 日志

INFO[2023-04-16 00:53:56] reading config file: data/config.json        
INFO[2023-04-16 00:53:56] config file not exists, creating default config file 
INFO[2023-04-16 00:53:56] load config from env with prefix:            
ERRO[2023-04-16 00:53:56] failed delete temp file: unlinkat /opt/alist/data/temp/aria2/temp/aria2: device or resource busy 
INFO[2023-04-16 00:53:56] init logrus...                               
INFO[2023-04-16 00:53:56] Successfully created the admin user and the initial password is: Xp5Gtyok 
INFO[2023-04-16 00:53:57] start server @ 0.0.0.0:5244                  
INFO[2023-04-16 00:53:57] qbittorrent not ready.                       
INFO[2023-04-16 01:02:51] reading config file: data/config.json        
INFO[2023-04-16 01:02:51] config file not exists, creating default config file 
INFO[2023-04-16 01:02:51] load config from env with prefix:            
ERRO[2023-04-16 01:02:51] failed delete temp file: unlinkat /opt/alist/data/temp/aria2/temp/aria2: device or resource busy 
INFO[2023-04-16 01:02:51] init logrus...                               
INFO[2023-04-16 01:02:51] Successfully created the admin user and the initial password is: 3I5butA3 
INFO[2023-04-16 01:02:51] start server @ 0.0.0.0:5244                  
INFO[2023-04-16 01:02:51] qbittorrent not ready.

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by PWMFI** [comment]
Fedora OS 用docker部署

**Comment by PWMFI** [comment]
docker run --privileged -d --restart=always -v /etc/alist:/opt/alist/data -p 5244:5244 -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist" xhofe/alist-aria2:latest

**Reply by firimar** [reply]
创建一个volume ，把 -v /etc/alist:/opt/alist/data 中的 /etc/alist 换成volume地址

**Comment by xhofe** [comment]
提供一下配置文件。

**Comment by PWMFI** [comment]
> 提供一下配置文件。

什么的配置文件，我要去哪里找，目前状况就是，重启后容器就被重置一样

**Reply by xhofe** [reply]
容器内 /opt/alist/config.json

**Reply by PWMFI** [reply]
> 容器内 /opt/alist/config.json！[上传IMG_3368.jpeg...]()没有这个文件

**Comment by hululuuuuu** [comment]
mac 权限不足导致的
可以在当前mac用户目录下新建文件夹 例如： /Users/xxx当前用户/alist/alistConfig:/opt/alist/data
